tests/search/setup_search_tests.py

```diff
@@ -182,6 +188,24 @@ def _create_family_metadata(db: Session, family: VespaFixture):
     db.commit()
 
 
+def _create_family_metadata_deterministic(db: Session, family: VespaFixture):
+    metadata_values = defaultdict(list)
+    family_metadata = family["fields"]["metadata"]
+    if not family_metadata:
+        return
+    for metadata in family_metadata:
+        name = metadata["name"]  # type: ignore
+        value = metadata["value"]  # type: ignore
```

Comment:
Wondering what the linting issue is that is being ignored here? If we definitely need it it'd be good to add the reason

Reply:
suggestion: Not sure if this makes a linting difference, but super minor simplification if you are interested
```suggestion
        name, value = metadata.values()
```

Reply:
![image](https://github.com/user-attachments/assets/bcf33b36-aa17-4e37-a90c-2f18b2183d32)

For some reason the linter thinks metadata is a string

Reply:
Ah this is probably a failing of `VespaFixture`

Reply:
Recommend leaving it then in that case, we can resolve it another time